app/src/main/java/com/globant/example/mentorapp/home/presentation/model/UsersModel.java

```diff
@@ -0,0 +1,32 @@
+package com.globant.example.mentorapp.home.presentation.model;
+
+import java.util.List;
+
+/**
+ * Created by oscar.amezquita on 16/06/2017.
+ */
+
+public class UsersModel {
```

Comment:
If this model represents a ViewModel, I suggest to rename it to be more descriptive: HomeViewModel